events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -24,8 +26,8 @@
         TEN_SECONDS_MS = 10 * ONE_SECOND_MS,
         ZONE_COLOR_INDEX = 19,
         HALF_MULTIPLIER = 0.5,
-        FIRST_WAIT_TO_COUNT_AVATARS = 1000,
-        WAIT_TO_SHOW_QUESTION = 500,
+        FIRST_WAIT_TO_COUNT_AVATARS = ONE_SECOND_MS,
```

Comment:
have units in the variable name: 
FIRST_WAIT_TO_COUNT_AVATARS -->
FIRST_WAIT_TO_COUNT_AVATARS_MS